source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1663,9 +1666,14 @@ def _get_role(self):
 	def _get_name(self):
 		# #4364 On some versions of Windows name contains redundant information that is available either in the role or the value, however on Windows 10 Anniversary Update and later the value is empty, so we cannot simply dismiss the name.
 		if super(TrayClockWClass, self).value is None:
-			return super(TrayClockWClass, self).name
+			return super(TrayClockWClass, self).name.replace(CHAR_LTR_MARK,'').replace(CHAR_RTL_MARK,'')

```

Comment:
Coming at this fresh I think I would put the replacement on its own line, e.g. create a variable holding the name, then a chained `replace` combined with the return on the next line. I don't know if we recommend a maximum line length though, maybe I'm being fussy.